index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div class="landingPage">
+        <header>
+            <div class="logo">My Logo</div>
+            <div class="menu">
+                <img src="./images/menu.svg" class="menuIcon" alt="">
+            </div>
+        </header>
+        <div class="container">
+            <h1>I'm Shayan<br>Glad to see you!</h1>
+            <p class="LandingPageDetails" >I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <div class="connect">
+                <p class="letsConnect" >LETS CONNECT</p>
+                <div class="socialLinks">
+                    <a class="socialIcon" href="#">
+                        <img src="./images/twitter.svg" alt="Twitter">
+                    </a>
+                    <a class="socialIcon" href="#">
+                        <img src="./images/linkedin.svg" alt="Linkedin">
+                    </a>
+                    <a class="socialIcon" href="#">
+                        <img src="./images/Vector (1).svg" alt="unknown">
+                    </a>
+                    <a class="socialIcon" href="#">
+                        <img src="./images/github.svg" alt="Github">
+                    </a>
+                    <a class="socialIcon" href="#">
+                        <img src="./images/Vector (3).svg" alt="unknown">
+                    </a>
+                </div>
```

Comment:
- [ ] [REQUIRED] Nice work but the list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag ✔️ ). Please implement my suggestions 🙏🏼